test/uk/gov/hmrc/transitmovementsrouter/controllers/MessageControllerSpec.scala

```diff
@@ -165,6 +167,7 @@ class MessageControllerSpec
       mockSDESService,
       mockInternalAuthActionProvider,
       mockStatusMonitoringService,
+      mockAuditingService,
```

Comment:
can not see any test cases to verify the auditing call with messagetType payload. Please add them.

Reply:
Agree. Will add verification of audit service calls.